sysutils/bash-completion/files/patch-xine-add-more-formats.diff

```diff
@@ -4,6 +4,7 @@ Origin: vendor
 Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/bash-completion/+bug/311525
 Bug-Debian: https://bugs.debian.org/496162
 
+Updated for version 2.12 for MacPorts
```

Comment:
But this is for version 2.13, isn't it?

Reply:
The patches were changed [here](https://github.com/macports/macports-ports/pull/22795) and you said back then we should indicate they were no longer taken directly from another distribution. Hence I added they were changed for 2.12.